packages/widget/src/components/address-input/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { css } from 'lit';
+
+export const styles = css`
+  .inputAddressSection {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 0.5rem;
+    min-height: 7.75rem;
```

Comment:
these `min-height` props (here and next class) are used only for spacing between elements which is pretty wrong. i would remove them and set appropriate gap/margin.

no need to change this now (so PR can be merged), but please then add `TODO:`, so I/we can update CSS in a day or two.

Reply:
we can do that together in a call, so I can explain a bit further the approach with no setting fixed values fro height/width

Reply:
adding gap on the form using flex could be a solution